<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgServerRecordResult</class>
 <widget class="QDialog" name="DlgServerRecordResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>云端录制</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgServerRecordResult {
	background: rgb(255,255,255);
}
QPushButton {
	background:rgb(57,129,252);
	color: rgb(255,255,255);
	border: 1px solid rgb(57,129,252);
	border-radius: 2px;
	min-height: 24px;
	min-width: 60px;
}
QPushButton:hover, QPushButton:pressed {
	background: rgb(5,117,197);
}
#recordPath {
	min-height: 24px;
	border: 1px solid rgb(240,240,240);
	border-radius: 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">#DlgServerRecordResult {
	background: rgb(255,255,255);
}
QPushButton {
	background:rgb(57,129,252);
	color: rgb(255,255,255);
	border: 1px solid rgb(57,129,252);
	border-radius: 2px;
	min-height: 24px;
	min-width: 60px;
}
QPushButton:hover, QPushButton:pressed {
	background: rgb(5,117,197);
}</string>
     </property>
     <property name="text">
      <string>录像地址：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCopy">
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="recordPath"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
